client/src/App.tsx

```diff
@@ -1255,6 +1263,16 @@ function App() {
 					onCreate={onCreateRoom}
 				/>
 			</Modal>
+			<Modal
```

Comment:
The above two JSX elements are not allowed, I need to wrap them into <div> to make it as a single element. However, it shows a border around the Modal when displayed. What should I do

Reply:
instead of using an empty <div> as the wrapper you can wrap in what's called a react fragment,
either done like <React.Fragment> ... </React.Fragment> or even simpler <> ... </> which is basically an empty container, this way we dont apply styles